index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather Repawrt</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css" />
+</head>
+
+<body>
+    <header class="header__header">
+        <h1>JAY PUG Tells the Weather</h1>
+        <span>For the pawsome city of 
+            <span id="headerCityName" class="header__city-name"></span></span>
```

Comment:
this works fine! something more common you might see is this:

```suggestion
            <h3>For the pawsome city of 
                <span id="headerCityName" class="header__city-name"></span>
            </h3>
```

nested `span` elements don't occur often. Usually you would pick a `p` or heading element, then nest the `span` inside!